### PR TITLE
test: fix jasmine tests

### PR DESCRIPTION
**Pull request by tancnle**
# Description

This commit fixes dependencies issues that cause

- `npm install` to fail 
- `ng test` does not pick up any specs

As part of this PR, I have also:

- Fix all 16 failing tests 
- Ensure CI run these unit tests

See original [PR](https://github.com/thoth-tech/doubtfire-web/pull/6).

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

1. Checkout this branch
2. Install the required dependencies `npm install`
3. Run the tests `npm test`

Screen Shot 2022-04-17 at 6 15 39 pm

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

**Comment by tancnle** [comment]
👋🏼 @macite @jakerenzella could you kindly review this PR, please? 🙏🏼

**Comment by tancnle** [comment]
Converted to draft to investigate CI failure.

**Comment by tancnle** [comment]
CI failure has been addressed. This PR is ready for review 👍🏼.

**Comment by tancnle** [comment]
@jakerenzella Any thoughts on this PR? 🙏🏼

File: src/app/api/models/user/user.service.spec.ts [hunk 1]
```diff
@@ -89,7 +96,7 @@ describe('UserService', () => {
     tick();
   }));
 
-  it('should delete a user', fakeAsync(() => {
+  xit('should delete a user', fakeAsync(() => {
```
Comment (jakerenzella) [inline src/app/api/models/user/user.service.spec.ts:99]: Is `xit` intentional?

**Review by jakerenzella — COMMENTED** [review]
Looks good, one quick qn

**Review by jakerenzella — APPROVED** [review]
Nevermind, approved!

File: src/app/api/models/user/user.service.spec.ts (hunk 1 above)
Comment (tancnle) [reply src/app/api/models/user/user.service.spec.ts:99]: Yes, that's intentional. We can remove the test altogether if needed.

**Pull request merged**